bin/gather_pipelines

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Run this script to collect openshift-pipelines related debug information
+
+set -Euox pipefail
```

Comment:
I think you meant -e not -E : 

```
             -E      If set, any trap on ERR is inherited by shell functions, command substitutions, and commands executed in a subshell environment.  The ERR trap
                      is normally not inherited in such cases.
```

I dont think any trap

and `"-o` pipefail"